runelite-client/src/main/java/net/runelite/client/plugins/statusorbs/StatusOrbsPlugin.java

```diff
@@ -58,6 +70,39 @@
 )
 public class StatusOrbsPlugin extends Plugin
 {
+	private static final ImmutableSet<Integer> ALL_GRACEFUL_HOODS = ImmutableSet.of(
```

Comment:
Looks like this comment is missing [from the original](https://github.com/runelite/runelite/pull/6520/commits/66e5b8aa608d0a8806fc03f66d09ae057952da73#diff-16890d9c8334c0901971acbf4f7b5de5L57)?

```
// TODO It would be nice if we have the IDs for just the equipped variants of the Graceful set items.
```

Not a big deal, but we should make sure that's not forgotten.